src/test/java/com/projetoGerenciamentoCurso/GerenciamentoCurso/Controller/AlunoControllerTest.java

```diff
@@ -13,14 +15,15 @@
 
 @SpringBootTest
 @AutoConfigureMockMvc
+@FixMethodOrder(MethodSorters.JVM)
 public class AlunoControllerTest {
 
 	
 	@Autowired
 	private MockMvc mockMvc;
 	
 	@Test
-	private void criarAlunoRetornaSucesso() throws Exception {
+	public void criarAlunoRetornaSucesso() throws Exception {

```

Comment:
Retirar o "public" de todas as classes de teste, pois isso está implícito no JUnit 5, como foi pedido na última revisão
```suggestion
	void criarAlunoRetornaSucesso() throws Exception {
```

Reply:
feito